bench/src/bulk.rs

```diff
@@ -3,10 +3,10 @@ use std::time::Instant;
 
 use anyhow::{anyhow, Context, Result};
 use futures::StreamExt;
-use tokio::runtime::current_thread::Runtime;
 use tracing::trace;
 
-fn main() {
+#[tokio::main(threaded_scheduler)]
```

Comment:
This still needs to be using the single-threaded runtime, with the client and server on distinct threads.

Reply:
I found this: https://github.com/tokio-rs/tokio/issues/1846.
It seems to be easier to match the performance with using the threaded scheduler, I can still make it single threaded if you wish.

Reply:
The real goal for this tool is to make it easy to profile quinn itself, and using exactly two threads with no inter-thread communications means fewer variables. I don't expect it to be extremely difficult to move the server's `Endpoint` construction inwards a little, so that seems like an easy win to me.

This could be accomplished by constructing a `std::net::UdpSocket` to get the local address from and passing that into the spawned thread for use with `EndpointBuilder::with_socket`, or by switching the client/server around so the server is constructed on the main thread, its address fetched, and that address passed to the spawned thread for the client to connect to once constructed.

Reply:
I solved it differently. As far as I understood the whole performance issue arose because of the runtime splitting between the server and client, so I just moved the runtime that was used to get the socket to the server and made a new runtime for the client.